CHANGELOG.md

```diff
@@ -6,10 +6,11 @@ All notable changes to this project will be documented in this file.
 
 ### Added
 
-### Changed 
+### Changed
 
 ### Fixed
 - Fixed text wrapping problem in the block explorer
+- Fixed displaying memo byte string as CBOR integer, when it contains arbitrary bytes.
```

Comment:
Are we sure this issue is limited to integers being shown? Perhaps the message should be a bit more general.

Reply:
I actually check this for both CBOR int and CBOR string, so I'll correct the changelog to be more precise